package.json

```
@@ -27,7 +27,7 @@
     "build-storybook": "nx build-storybook designsystem",
     "chromatic": "chromatic",
     "publish": "node ./scripts/publish.js",
-    "postinstall": "npm run transpile:tools && npm run extract-scss-variables && npm run generate-design-tokens && npx nx build core && npm run transpile:generate-mocks",
+    "postinstall": "npm run transpile:tools && npm run extract-scss-variables && npm run generate-design-tokens && npx nx build core && npx nx build angular && npm run transpile:generate-mocks",
```

```suggestion
    "postinstall": "npm run transpile:tools && npm run extract-scss-variables && npm run generate-design-tokens && npx nx build core && npm run transpile:generate-mocks",
```

Just a note to self here - we should remove this and find a better solution before merge.